ClientApp/src/app/forgotpassword/forgotpassword.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { ResetPasswordService } from '../server-api/reset-password.service';
+
+@Component({
+  selector: 'app-forgotpassword',
+  templateUrl: './forgotpassword.component.html',
+  styleUrls: ['./forgotpassword.component.css']
+})
+export class ForgotpasswordComponent implements OnInit {
```

Comment:
Could you capitalize the name of this class to `ForgotPasswordComponent`? For consistency with the rest of them